docs-new/docs/graphql/core/databases/ms-sql-server/getting-started/cloud.mdx

```diff
@@ -0,0 +1,146 @@
+---
+description: Hasura Cloud for MS SQL Server
+keywords:
+  - hasura
+  - docs
+  - databases
+  - ms sql
+  - sql server
+  - hasura-cloud
+---
+
+import Thumbnail from "@site/src/components/Thumbnail/Thumbnail";
+
+# Get Started with Hasura Cloud & MS SQL Server {#database-ms-sql-server-cloud}
```

Comment:
Custom Id is not required for the top level h1 heading. as any reference can directly be linked to file itself.